tests/framework/installer/ceph_manifests.go

```diff
@@ -3769,6 +3779,8 @@ spec:
   mirroring:
     enabled: true
     mode: image
+  quotas:
+    enabled: true
```

Comment:
How about setting values for both the quotas here in the test.

Reply:
I wasn't sure what would be a proper value. Is any data added to the pool during testing? If so, what would be a proper quota?

Reply:
We only write a few files in the tests, so just set it something like a few GB and 1000 objects so we don't hit the limit.

Reply:
Added. I went with 1000000 for objects though as rbd uses small rados objects.